app/services/association_authorizer.rb

```diff
@@ -1,20 +1,21 @@
 class AssociationAuthorizer
-  def self.authorized_associations(associations,  klass_name = nil, should_raise_exception = true)
+  def self.authorized_associations(associations,  klass_name = nil, should_raise_exception = true, action = 'view')
```

Comment:
Oh sorry, you meant specifically the action piece.  It didn't make sense to me to make a service with a generic name that only supports one permission type.

I'll revert it if you want and hardcode view again.